examples/cohort_replicator_kafka_pg/examples/cohort_replicator_kafka_pg.rs

```diff
@@ -0,0 +1,100 @@
+use std::sync::Arc;
+
+use async_trait::async_trait;
+use cohort_banking::{
+    callbacks::statemap_installer::StatemapInstallerImpl,
+    state::postgres::{database::Database, database_config::DatabaseConfig},
+};
+use talos_certifier::{env_var, env_var_with_defaults, ports::MessageReciever};
+use talos_certifier_adapters::{KafkaConfig, KafkaConsumer};
+use talos_cohort_replicator::{talos_cohort_replicator, CohortReplicatorConfig, ReplicatorSnapshot};
+
+use cohort_banking::state::postgres::database::DatabaseError;
+use tokio::{signal, try_join};
+
+pub static SNAPSHOT_SINGLETON_ROW_ID: &str = "SINGLETON";
+
+pub struct SnapshotApi {
+    db: Arc<Database>,
+}
+
+#[async_trait]
+impl ReplicatorSnapshot for SnapshotApi {
+    async fn get_snapshot(&self) -> Result<u64, String> {
+        let client = &self.db.get().await.unwrap();
+
+        let sql = r#"SELECT "version" FROM cohort_snapshot WHERE id = $1"#;
+
+        let _prepare = client
+            .prepare_cached(sql)
+            .await
+            .map_err(|e| DatabaseError::prepare(e.to_string(), sql.to_string()).to_string())?;
+        let snapshot_row = client.query_one(sql, &[&SNAPSHOT_SINGLETON_ROW_ID]).await.map_err(|e| e.to_string())?;
+
+        let snapshot: i64 = snapshot_row.get("version");
+
+        Ok(snapshot as u64)
+    }
+}
+
+#[tokio::main]
+async fn main() {
+    env_logger::builder().format_timestamp_millis().init();
+
+    // 0. Create required items.
+    //  a. Create Kafka consumer
+    let mut kafka_config = KafkaConfig::from_env();
+    kafka_config.group_id = "talos-replicator-dev-1".to_string();
```

Comment:
Why not from config?

Reply:
Thanks for noticing it, I thought I updated this, must have been in some experimental commit or stash. I will update this.